dotcom-rendering/package.json

```diff
@@ -28,7 +28,7 @@
     "cypress:run": "cypress run",
     "cypress:run:e2e": "cypress run --spec 'cypress/integration/e2e/**/*'",
     "cypress:run:mocked": "cypress run --spec 'cypress/integration/mocked/**/*'",
-    "makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/webpack/frontend.js"
+    "makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/frontend/dev-server.js"
```

Comment:
I think this should be going to ./scripts/webpack/webpack.config.js

Reply:
yeah i was too vociferous with my pasting. hasty paster